sciml/op_lib/nucleation.py

```diff
@@ -0,0 +1,145 @@
+import h5py as h5
+import numpy as np
+from scipy.stats import qmc
+import matplotlib.pyplot as plt
+
+DX = 0.03125 # Grid spacing in FlashX simulations
+
+def heater_init(xmin, xmax, num_sites):
+    r"""
+    Initialize the nucleation sites on the 1-D heater. Returns a 1D line of sites
+    that is dependant on wall temperature.
+
+    Args:
+        xmin (float): The x-coordinate where heater begins.
+        xmax (float): The x-coordinate where heater ends.
+        num_sites (int): The number of nucleation sites in the domain.
+
+    Returns:
+        numpy.ndarray: The coordinates of the heater nucleation sites.
+    """
+    x_sites = np.ndarray(num_sites, dtype=float)
+    y_sites = np.ndarray(num_sites, dtype=float)
+
+    halton_sequence = qmc.Halton(d=2, seed=1)
```

Comment:
Will the halton sequence match flash-x?  I think we need to be careful about that... This is fine for testing, but maybe it would be better to just read the heater initialization / nucleation site locations from simulation output